README.md

```diff
@@ -88,26 +88,22 @@ The steps required are outlined below.
 - Run the build script appropriate for your environment.
      - **Windows:** [build.cmd](https://github.com/dotnet/templating/blob/master/build.cmd)
      - **Mac/Linux**: [build.sh](https://github.com/dotnet/templating/blob/master/build.sh) 
-- When running the build script, the existing built-in command `dotnet new` will be preserved. A new command `dotnet new3` will be enabled which allows you to create
-files with the latest Template Engine.
-- That's it! Now you can run `dotnet new3`.
+- When running the build script, the existing built-in command `dotnet new` will be preserved. To run `dotnet new3` run `dotnet <your repo location>\artifacts\bin\dotnet-new3\Debug\netcoreapp2.1\dotnet-new3.dll` (root path to `dotnet-new3.dll` is skipped in all commands below).
```

Comment:
A comma after new3 would be nice like
```
. To run `dotnet new3`, run `dotnet...`
```

Reply:
Did we remove this functionality or was it already broken? It looks like an easy and nice way to test your changes. Is bringing it back to life hard?

Reply:
@bozturkMSFT I was looking at this yesterday. It seems that the functionality was lost about 2 years ago when moving to arcade build scripts. I can ask the question around, I would say it should be possible but need a bit of digging into how arcade build scripts work and how we can do custom actions there.

Reply:
Created https://github.com/dotnet/templating/issues/2521 for this